Formula/erlang.rb

```diff
@@ -40,13 +40,13 @@ class Erlang < Formula
   fails_with :llvm
 
   resource "man" do
-    url "https://www.erlang.org/download/otp_doc_man_18.3.tar.gz"
-    sha256 "978be100e9016874921b3ad1a65ee46b7b6a1e597b8db2ec4b5ef436d4c9ecc2"
+    url "http://erlang.org/download/otp_doc_man_19.0.tar.gz"
```

Comment:
@tmjoen Yes, you are right. I believe that we no longer need to contact them.

I was checking the `https` and it redirects to `http` version:

```
$ curl -I https://www.erlang.org/download/otp_doc_man_19.0.tar.gz
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Wed, 22 Jun 2016 21:33:34 GMT
Content-Type: text/html
Content-Length: 184
Connection: keep-alive
Location: http://erlang.org/download/otp_doc_man_19.0.tar.gz
```

In chrome, my https file checksum is different from http file. But in another browsers and curl I don't have this problem.

I delete my Erlang version and update the formula to use the `https` and it works.

@tmjoen, @DomT4 Could you checkout the branch and test it too?

```bash
~/src/personal/homebrew-core 2.3.0 (ek-erlang-19 * u=)
$ g d
diff --git a/Formula/erlang.rb b/Formula/erlang.rb
index d20c281..f1a2d79 100644
--- a/Formula/erlang.rb
+++ b/Formula/erlang.rb
@@ -40,12 +40,12 @@ class Erlang < Formula
   fails_with :llvm

   resource "man" do
-    url "http://erlang.org/download/otp_doc_man_19.0.tar.gz"
+    url "https://www.erlang.org/download/otp_doc_man_19.0.tar.gz"
     sha256 "c7a3d6d85a5a2b96d844297a3fa1bee448c3dd86237734688466249fd5a1401e"
   end

   resource "html" do
-    url "http://erlang.org/download/otp_doc_html_19.0.tar.gz"
+    url "https://www.erlang.org/download/otp_doc_html_19.0.tar.gz"
     sha256 "b6f7c4e964673333f6c3eea8530dd988b41339b8912ae293f6f1b429489159ff"
   end


~/src/personal/homebrew-core 2.3.0 (ek-erlang-19 * u=)
$ brew install Formula/erlang.rb
==> Downloading https://homebrew.bintray.com/bottles/erlang-19.0.el_capitan.bottle.2.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "erlang"
Download failed: https://homebrew.bintray.com/bottles/erlang-19.0.el_capitan.bottle.2.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/erlang/otp/archive/OTP-19.0.tar.gz
==> Downloading from https://codeload.github.com/erlang/otp/tar.gz/OTP-19.0
######################################################################## 100.0%
==> ./otp_build autoconf
==> ./configure --disable-silent-rules --prefix=/opt/boxen/homebrew/Cellar/erlang/19.0 --enable-kernel-poll --enable-threads --enable-sctp --enable-dynamic-ss
==> make
==> make install
==> Downloading https://www.erlang.org/download/otp_doc_man_19.0.tar.gz
==> Downloading from http://erlang.org/download/otp_doc_man_19.0.tar.gz
######################################################################## 100.0%
==> Downloading https://www.erlang.org/download/otp_doc_html_19.0.tar.gz
==> Downloading from http://erlang.org/download/otp_doc_html_19.0.tar.gz
######################################################################## 100.0%
==> Caveats
Man pages can be found in:
  /opt/boxen/homebrew/opt/erlang/lib/erlang/man

Access them with `erl -man`, or add this directory to MANPATH.
==> Summary
🍺  /opt/boxen/homebrew/Cellar/erlang/19.0: 7,292 files, 278.6M, built in 12 minutes 40 seconds
```